core/controllers/questions_list.py

```diff
@@ -97,7 +101,7 @@ def get(self, comma_separated_skill_ids):
 
         self.values.update({
             'question_summary_dicts': return_dicts,
-            'next_start_cursor': next_start_cursor
+            'next_start_cursor': offset
```

Comment:
```suggestion
            'offset;': offset
```
but the frontend will need to be updated too.

Reply:
Maybe next_offset? And agree re updating the frontend.

Reply:
Done.